airflow/providers/common/sql/operators/sql.py

```diff
@@ -221,6 +220,16 @@ def execute(self, context):
 
         return output
 
+    def on_kill(self) -> None:
+        self.log.info("Stopping:", self._hook.query_ids)
+
+        results = []
+        for query_id in self._hook.query_ids.copy():
+            result = self._hook.stop_query(query_id)
+            results.append(result)
+
+        self.log.info("Termination successful:", results)
```

Comment:
not necessarily.
query may have already completed successfully in the time between so we don't really know if we succeeded in termination we know only that we requested termination.

Reply:
You're right. But the current solution for Trino tries to stop query anyway
https://github.com/apache/airflow/blob/9337aa92c082db36e82eb314585591394fe8ff27/airflow/providers/trino/operators/trino.py#L62

Reply:
I meant to consider just changing the log entry from successful to requested?